### ISSUE TITLE
No wallet backup was found.

### ISSUE THREAD

**Issue by chrisdouglas** [bitcoin-wallet/bitcoin-wallet#644]
I backed up my wallet a couple weeks ago in preparation for upgrading my phone. (bitoin-wallet-backup-2020-09-28-18-21) Now with the wallet app reinstalled when I try to restore the wallet I get the following message:

_RESTORE WALLET
No wallet backup was found.
Make sure it is placed in external storage (/Download).
..._

I've copied the wallet backup file to both the phone's Download directory and the SD card's Download directory and I still get the same error.

Where does the app expect the wallet to be? How do I fix this?

Thank you,
Chris

**Comment by chrisdouglas** [comment]
I found this: https://bitcoin.stackexchange.com/a/50534

I had installed version 7.27, the newest version available on F-Droid. I had apparently backed up the wallet using version 7.22, the newest version available from F-Droid that would install on my phone before it was upgraded from Android 9 to Android 10. I guess that there's incompatibilities in wallets between the two versions such that I couldn't restore the wallet using 7.27. After downgrading to 7.22 I was able to restore the wallet and then upgrade to 7.27.
